### DISCUSSION TITLE
React

### DISCUSSION THREAD

**Discussion by dinaerteneto** [balta-io/forum#146] [Tira Dúvidas] [answered]
Aparentemente entre as aulas: useEffect e ReactRouter faltou ou faltaram vídeos.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @dinaerteneto, realmente... abri um chamado interno aqui para verificarmos.

**Reply by amarinafroes** [reply]
Olá, aparentemente o erro ainda não foi solucionado, poderia me informar se a alguma previsão para a solução desse problema?
